test/com/facebook/buck/testrunner/testdata/simple_testng/test/BUCK.fixture

```diff
@@ -1,7 +1,25 @@
 java_test(
-  name = 'test',
+  name = 'SimpleTest',
```

Comment:
This name doesn't follow naming convention we usually use, it should be lower case and words separated by `-`.

Reply:
Changed